### PR TITLE
mgmt support azure-resourcemanager-cdn

### PR DESCRIPTION
**Pull request by xseeseesee**

### CODE REVIEW

**Comment by weidongxu-microsoft** [comment]
you need to add back the sample, else there is no test.

**Comment by xccc-msft** [comment]
> you need to add back the azure/sample, else there is no test.

Just open for code changes review. I'm trying to add some new tests and checking the comments if any fix required inside impls.

**Review by weidongxu-microsoft — APPROVED** [review]

File: sdk/resourcemanager/azure-resourcemanager-cdn/src/main/java/com/azure/resourcemanager/cdn/implementation/CdnEndpointImpl.java [hunk 1]
```diff
@@ -134,25 +132,34 @@ public String id() {
...
                 itemToDelete.name()
-            ), 32, 32)
-            .last();
+            ), 32, 32);
 
-        return Flux.zip(originUpdateTask, endpointUpdateTask, customDomainTask)
-            .last()
-            .map(objects -> {
-                self.setInner(objects.getT2());
+        Flux<CustomDomainInner> customDomainTask = Flux.concat(customDomainCreateTask, customDomainDeleteTask);
```
Comment (ChenTanyi) [inline sdk/resourcemanager/azure-resourcemanager-cdn/src/main/java/com/azure/resourcemanager/cdn/implementation/CdnEndpointImpl.java:152]: If no order needed, use `Flux.mergeDelayError`

File: sdk/resourcemanager/azure-resourcemanager-cdn/src/main/java/com/azure/resourcemanager/cdn/implementation/CdnEndpointImpl.java [hunk 2]
```diff
@@ -134,25 +132,34 @@ public String id() {
...
+
+        return customDomainTask.then(originUpdateTask).then(endpointUpdateTask)
```
Comment (ChenTanyi) [inline sdk/resourcemanager/azure-resourcemanager-cdn/src/main/java/com/azure/resourcemanager/cdn/implementation/CdnEndpointImpl.java:154]: If no order needed, use `Flux.mergeDelayError(customDomainTask.then(), originUpdateTask.then(), endpointUpdateTask)`, for parallel

File: sdk/resourcemanager/azure-resourcemanager-cdn/src/main/java/com/azure/resourcemanager/cdn/implementation/CdnEndpointImpl.java (hunk 1 above)
Comment (xccc-msft) [reply sdk/resourcemanager/azure-resourcemanager-cdn/src/main/java/com/azure/resourcemanager/cdn/implementation/CdnEndpointImpl.java:152]: The concern to have concat came from [Azure/azure-libraries-for-net#891](https://github.com/Azure/azure-libraries-for-net/issues/891). For safe, I just concat the custom domain tasks. For tasks of different resources, we can use merge.

File: sdk/resourcemanager/azure-resourcemanager-cdn/src/main/java/com/azure/resourcemanager/cdn/implementation/CdnEndpointImpl.java (hunk 2 above)
Comment (xccc-msft) [reply sdk/resourcemanager/azure-resourcemanager-cdn/src/main/java/com/azure/resourcemanager/cdn/implementation/CdnEndpointImpl.java]: Updated.

**Review by ChenTanyi — APPROVED** [review]

**Pull request merged**